rhio/src/node/actor.rs

```diff
@@ -391,6 +411,13 @@ impl NodeActor {
                     return Ok(());
                 }
 
+                metrics::counter!(
+                    MESSAGE_RECEIVE_TOTAL,
+                    LABEL_SOURCE => LABEL_SOURCE_NETWORK,
+                    LABEL_MSG_TYPE => LABEL_NETWORK_MSG_TYPE_NATS_MESSAGE,
+                    LABEL_REMOTE_BUCKET => subject.to_string()
```

Comment:
Nats messages aren't connected to a s3 bucket instance, they contain domain specific messages. Considering this I'm not sure that `LABEL_REMOTE_BUCKET` should be present here.